media-gfx/openvdb/openvdb-5.2.0-r1.ebuild

```diff
@@ -1,26 +1,29 @@
 # Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
 
-PYTHON_COMPAT=( python3_6 )
+PYTHON_COMPAT=( python3_{7,8} )
 
-inherit cmake-utils flag-o-matic python-single-r1
+inherit cmake flag-o-matic python-single-r1
 
-DESCRIPTION="Libs for the efficient manipulation of volumetric data"
+DESCRIPTION="Library for the efficient manipulation of volumetric data"
 HOMEPAGE="https://www.openvdb.org"
-SRC_URI="https://github.com/dreamworksanimation/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+SRC_URI="https://github.com/AcademySoftwareFoundation/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
 
 LICENSE="MPL-2.0"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE="+abi4-compat doc python test"
+IUSE="abi3-compat abi4-compat abi5-compat doc python test"
 RESTRICT="!test? ( test )"
-REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} )"
+REQUIRED_USE="
+	python? ( ${PYTHON_REQUIRED_USE} )
+	^^ ( abi3-compat abi4-compat abi5-compat )
+"
 
 RDEPEND="
-	>=dev-libs/boost-1.62:=
-	>=dev-libs/c-blosc-1.5.0
+	dev-libs/boost:=
+	dev-libs/c-blosc
```

Comment:
If things provide a subslot and we link against them, subscribe to it with :=. c-blosc has one, check all other deps too?

Reply:
I have been through all the ebuilds and added them for c-blosc, jemalloc, log4cplus. I removed the subscription for glfw as it does not have a slot operator.